music_service_async_interface/__init__.py

```diff
@@ -112,11 +113,10 @@ async def get_file_url(self, *args, **kwargs) -> str:
         ...
 
     if AsyncSeekableHTTPFile is not None:
+
         async def get_async_file(self, filename: Optional[str] = None, *args, **kwargs) -> AsyncSeekableHTTPFile:
             return await AsyncSeekableHTTPFile.create(
-                await self.get_file_url(*args, **kwargs),
-                self.title if filename is None else filename,
-                self.sess.sess
+                await self.get_file_url(*args, **kwargs), self.title if filename is None else filename, self.sess.sess
```

Comment:
uhm, I liked previous split, it's much more readable

Reply:
Ok, now I know we just need a trailing coma to make this work as I want it to.
Could you please add it to this PR?